pagerduty/resource_pagerduty_incident_workflow_trigger.go

```diff
@@ -128,6 +128,11 @@ func resourcePagerDutyIncidentWorkflowTriggerDelete(ctx context.Context, d *sche
 }
 
 func validateIncidentWorkflowTrigger(_ context.Context, d *schema.ResourceDiff, _ interface{}) error {
+	oldType, newType := d.GetChange("type")
+	if oldType != nil && oldType != "" && oldType != newType {
+		return fmt.Errorf("trigger type cannot be changed")
+	}
+
```

Comment:
Did a quick spot check of the trigger stuff while I was here and saw that the config allowed you to change the type, but that change is ignored by the server. Instead of having a diff constantly appear and never be applied, we reject changing the type after the trigger has been created.